queries.sql

```diff
@@ -6,3 +6,10 @@ SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered = 't';
 SELECT * from animals WHERE name <> 'Gabumon';
 SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
```

Comment:
- [ ] Good job on executing transactions and taking screenshots for each one of them but please add the transactions instruction in your `queries.sql` file.